db/seeds.rb

```diff
@@ -11,11 +11,12 @@
 require 'nokogiri'
 require 'open-uri'
 
+User.create(id: 1, email: 'test@test.by', password: '12345')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.